### DISCUSSION TITLE
[Request] Add all mentionned accounts when replying to a post

### DISCUSSION THREAD

**Discussion by xylolink** [hyperspacedev/hyperspace#183] [Ideas]
**Is your feature request related to a problem? Please describe.**
Replying to a post only mentionned its author, not the accounts mentionned in the replied post.

**Describe the solution you'd like**
Automatically add to the reply every account involved in the replied post.

**Describe alternatives you've considered**
linked to #182 : typing "@" suggests accounts and in priority the previously mentionned ones

**Additional context**
None

**Comment by alicerunsonfedora** [comment]
We can definitely take a look at implementing something like this in a future feature update after v1.1.0.
